features/mbedtls/platform/inc/platform_mbed.h

```diff
@@ -21,6 +21,10 @@
 #define MBEDTLS_ENTROPY_HARDWARE_ALT
 #endif
 
+#if defined(MBED_CONF_FILESYSTEM_PRESENT)
```

Comment:
Where is it documented what this option entails? Does it ensure the presence of FS IO functions that Mbed TLS is using, with the same (standard) signatures?